src/ui/pages/Overview/Banners/Banners.tsx

```diff
@@ -75,8 +79,16 @@ export function Banners({ address }: { address: string }) {
           />
           <Spacer height={24} />
         </>
-      ) : null}
-      {ENABLE_DNA_BANNERS && !invitationBannerVisible ? (
+      ) : !preferences?.exploreZeroBannerDismissed ? (
+        <>
+          <ExploreZeroBanner
+            onDismiss={() =>
+              setPreferences({ exploreZeroBannerDismissed: true })
+            }
+          />
+          <Spacer height={24} />
+        </>
+      ) : ENABLE_DNA_BANNERS ? (
```

Comment:
May be it will be the time to remove these dna banners soon. 
I am pretty sure we will not return them, cause there is no more dna on Ethereum :))